services/api-server/src/simcore_service_api_server/utils/client_base.py

```diff
@@ -30,36 +34,82 @@ async def is_responsive(self) -> bool:
             resp.raise_for_status()
             return True
         except (httpx.HTTPStatusError, httpx.RequestError) as err:
-            log.error("%s not responsive: %s", self.service_name, err)
+            _logger.error("%s not responsive: %s", self.service_name, err)
             return False
 
     ping = is_responsive  # alias
 
 
+class _AsyncClientWithCaptures(httpx.AsyncClient):
```

Comment:
Just a suggestion: to me this sounds better. What does `captures` (plural) refer to? It is not not that clear.
```suggestion
class _AsyncCapturingClient(httpx.AsyncClient):
```

Reply:
I renamed it differently.